ADAL/src/telemetry/ADTelemetryPiiRules.m

```diff
@@ -24,26 +24,30 @@
 #import "ADTelemetryPiiRules.h"
 #import "ADTelemetryEventStrings.h"
 
-static NSDictionary *_piiRules;
+static NSSet *_piiRules;
 
 @implementation ADTelemetryPiiRules
 
 + (void)initialize
 {
-    _piiRules = @{AD_TELEMETRY_KEY_USER_ID: @YES};
+    _piiRules = [[NSSet alloc] initWithArray:@[AD_TELEMETRY_KEY_TENANT_ID,
+                                               AD_TELEMETRY_KEY_USER_ID,
+                                               AD_TELEMETRY_KEY_DEVICE_ID,
+                                               AD_TELEMETRY_KEY_APPLICATION_NAME,
+                                               AD_TELEMETRY_KEY_LOGIN_HINT,
+                                               AD_TELEMETRY_KEY_CLIENT_ID,
+                                               AD_TELEMETRY_KEY_ERROR_DESCRIPTION,
+                                               AD_TELEMETRY_KEY_HTTP_PATH,
+                                               AD_TELEMETRY_KEY_REQUEST_QUERY_PARAMS,
+                                               AD_TELEMETRY_KEY_HTTP_ERROR_DOMAIN,
```

Comment:
I don't think AD_TELEMETRY_KEY_HTTP_ERROR_DOMAIN is PII

Reply:
removed